.gitignore

```diff
@@ -12,3 +12,23 @@ config.status
 Makefile.config
 HYPRE_config.h
 autom4te.cache
+src/TAGS
```

Comment:
What is this one?  I have not seen it before.  Would it be better in your global `.gitignore`?

Reply:
Hi Rob, this file is generated by `make tags`, its purpose is to tell emacs, or other editors, where to find function definitions. See [this page](https://ftp.gnu.org/old-gnu/Manuals/emacs/html_node/emacs_338.html#:~:text=To%20select%20a%20tags%20table,variable%20tags%2Dfile%2Dname%20.) for instructions on how to use it. I believe it makes sense to have it on `gitignore` since the contents of this file evolve with the library. But I'm fine with removing it :) let me know!

Reply:
I didn't even realize we had a `tags` target. :)  Makes sense to keep this in `.gitignore`.  Thanks!